setup.cfg

```diff
@@ -15,7 +15,10 @@ license_files = LICENSE
 [options]
 zip_safe = True
 packages = find:
-install_requires = numpy
+install_requires = 
+    numpy
+    sklearn
+    ase
```

Comment:
I would rather not depend on ase here, since it is not used in the main code, only for examples.